warmup.krun

```diff
@@ -21,8 +21,13 @@ else:
 
 JDK8_BIN = os.path.join(JDK8_HOME, "bin", "java")
 
-HEAP_LIMIT = 2097152  # K == 2Gb
-STACK_LIMIT = 8192  # K
+if sys.platform.startswith("linux"):
+    GCC_ENV = {"LD_LIBRARY_PATH": os.path.join(DIR, "work/gcc-inst/lib64")}
+else:
```

Comment:
This should probably be `elif sys.platform.startswith("openbsd")` and the actual `else` branch should raise a `NotImplementedException` or similar.